instruments/nas/client.go

```diff
@@ -55,6 +55,26 @@ func (c *Client) Get(instrumentId string) (*GetInstrumentResponse, error) {
 	return &response, nil
 }
 
+func (c *Client) GetBankAccountFieldFormatting(country string, currency string, query QueryBankAccountFormatting) (*GetBankAccountFieldFormattingResponse, error) {
```

Comment:
Lets rearrange this like this, so it doesn't exceed the max amount of chars per line:
```
func (c *Client) GetBankAccountFieldFormatting(
	country string, 
	currency string, 
	query QueryBankAccountFormatting,
) (*GetBankAccountFieldFormattingResponse, error) {
}
```

Reply:
👍